be/src/vec/functions/array/function_array_range.cpp

```diff
@@ -146,14 +144,6 @@ struct RangeImplUtil {
         for (int i = 0; i < 3; ++i) {
             argument_columns[i] =
                     block.get_by_position(arguments[i]).column->convert_to_full_column_if_const();
-            if (auto* nullable = check_and_get_column<ColumnNullable>(*argument_columns[i])) {
-                // Danger: Here must dispose the null map data first! Because
-                // argument_columns[i]=nullable->get_nested_column_ptr(); will release the mem
-                // of column nullable mem of null map
-                VectorizedUtils::update_null_map(args_null_map->get_data(),
```

Comment:
`arg_null_map` was updated in this line, and was used be code after this line, then you removed this logic. Why does this function still work?

Reply:
because use_default_implementation_for_nulls() is true, when meet null it will work on default implementation

Reply:
you should remove var `args_null_map` if this var not working.

Reply:
I thought that `args_null_map` is working, and it's just dont need to update_null_map.

Reply:
because in the following code, `args_null_map` is used as a function parameter. If it is deleted, unexpected errors may occur.

Reply:
`arg_null_map` will be updated in two ways:
1. outside this function, in the default implementation of nulls. so these code could be safely deleted
2. in the functions Impl, that is, it's own logic. so we need this var.